en/cryptography.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Cryptography
+table_of_contents: true
+---
+
+# Cryptography
```

Comment:
```suggestion
# Cryptography
```

MyST/Sphinx only use frontmatter in special cases, so you don't need the `title` or `table_of_contents` references.